src/index.rs

```diff
@@ -76,6 +76,42 @@ impl fmt::Display for Column {
     }
 }
 
+/// Window Dimensions
+///
+/// Newtype to avoid passing values incorrectly
```

Comment:
Not quite what I meant about the newtype. Maybe it will be less code anyhow not to newtype `Point`. Let's forget about this part.